tests/testsuite/serviceaccount.go

```diff
@@ -63,6 +64,9 @@ func createServiceAccounts() {
 	createServiceAccount(ViewInstancetypeServiceAccountName)
 	createClusterRoleBinding(ViewInstancetypeServiceAccountName, "instancetype.kubevirt.io:view")
 
+	createServiceAccount(MigrateServiceAccountName)
+	createRoleBinding(MigrateServiceAccountName, "kubevirt.io:migrate")
```

Comment:
We need to clean this SA in the `createServiceAccounts` function below.

Reply:
done, thanks